### PR TITLE
Spotless apply

### PR DESCRIPTION
**Pull request by kortemik**

### CODE REVIEW

File: src/main/java/com/teragrep/rlp_03/SocketPoll.java [hunk 1]
```diff
@@ -94,31 +93,25 @@ public SocketPoll(
...
-                                "isWritable <{}>",
-                        selectionKey,
-                        selectionKey.isValid(),
-                        selectionKey.isConnectable(),
-                        selectionKey.isAcceptable(),
-                        selectionKey.isReadable(),
-                        selectionKey.isWritable()
-                );
+                LOGGER
+                        .debug(
```
Comment (eemhu) [inline src/main/java/com/teragrep/rlp_03/SocketPoll.java:103]: Perhaps these logs with bunch of variables could be replaced with:
https://www.baeldung.com/slf4j-parameterized-logging
See "addKeyValue() approach" on section "5. Fluent logging"

Comment (eemhu) [reply src/main/java/com/teragrep/rlp_03/SocketPoll.java:103]: Think it would be a cleaner approach if possible.

**Review by eemhu — APPROVED** [review]
Looks fine to me otherwise.

**Review by kortemik — COMMENTED** [review]
ok for me too

**Pull request merged**